gcc doesn't like #pragma ident in /usr/include/kerberosv5/com_err.h and emits warning, 
so remove Werror flag

--- squid-3.5.4/acinclude/compiler-flags.m4.~1~	2015-05-05 12:46:29.661133959 +0300
+++ squid-3.5.4/acinclude/compiler-flags.m4	2015-05-05 12:48:45.572231111 +0300
@@ -134,7 +134,7 @@
  case "$squid_cv_compiler" in
   gcc) 
    squid_cv_cc_option_werror="-Werror" 
-   squid_cv_cxx_option_werror="-Werror" 
+   squid_cv_cxx_option_werror="" 
    squid_cv_cc_option_wall="-Wall"
    squid_cv_cc_option_optimize="-O3"
    squid_cv_cc_arg_pipe="-pipe"
